Documenting a new post-mortem process (#7942)
PostHog/posthog.com: contents/handbook/engineering/incidents.md

24-48 hours after an incident, we should have a quick sync meeting with a small group of people involved in the incident, plus Tim. If you raised the incident, you can schedule this. If you don’t want to or can’t, Tim is happy to take over at this point, just let him know.

> Incident.io will automatically create an incident analysis doc that you can paste into a PR. 

Reviewer: I don't think this is true. It was fine before as the key point was true
```suggestion
Before this meeting you should create a PR against the [incidents analysis repository](https://github.com/PostHog/incidents-analysis). You can use Incident.io's automatic analysis as a starting point. In general it is advised to post short point in time summaries of actions taken in the incident Slack channel so that you can more easily fill this out after the fact.
```

> - If a post-mortem had significant impact on customers (like flags not being available or where we caused data loss for a large majority of customers), we should sanitize and copy the post mortem into the public [post-mortems repository](https://github.com/PostHog/post-mortems)

Reviewer 2: Any data loss that's our fault should be communicated
```suggestion
- If a post-mortem had significant impact on customers (like data loss or flags not being available), we should sanitize and copy the post mortem into the public [post-mortems repository](https://github.com/PostHog/post-mortems)
```

24-48 hours after an incident, we should have a quick sync meeting with a small group of people involved in the incident, plus Tim. If you raised the incident, you can schedule this. If you don’t want to or can’t, Tim is happy to take over at this point, just let him know.

Incident.io will automatically create an incident analysis doc that you can paste into a PR. 

> - If the post-mortem involves confidential information, such as customer data, it should go in the private [incidents analysis repository](https://github.com/PostHog/incidents-analysis).
> - In most other situations we should be as transparent as possible and document post-mortems in the public [post-mortems repository](https://github.com/PostHog/post-mortems)

Reviewer 3: Disagree. We should only publicise a post mortem if there was a noticeable effect on customers, like causing them downtime, or data loss. Overall transparency is good, but we don't want to give people a whole roadmap to our infrastructure. It's also a lot of work to word post mortems in a way that's appropriate to be public.

```suggestion
- If a post-mortem had significant impact on customers (like flags not being available or where we caused data loss for a large majority of customers), we should sanitize and copy the post mortem into the public [post-mortems repository](https://github.com/PostHog/post-mortems)
```
